docs/pages/api.rst

```diff
@@ -22,11 +22,14 @@ fromCallable
 
 Create a collection from a callable.
 
-Signature: ``Collection::fromCallable(callable $callable, ...$parameters): Collection;``
+.. tip:: This can be very useful when working with a PHP `Generator`_, since it will allow the collection
+    object to behave as if the Generator was rewindable.
+
+Signature: ``Collection::fromCallable(callable $callable, iterable $parameters = []): Collection;``
```

Comment:
don't forget to update these when changing method signatures 😬 
https://github.com/loophp/collection/pull/146

Reply:
Oops! Thanks!